044-Stable-identities-for-Kafka-Connect-worker-nodes.md

```diff
@@ -0,0 +1,176 @@
+* Enabling of the `StableConnectIdentities` feature gate in Strimzi version which supports it
+
+The migration will happen automatically and will use the following steps:
+* If the user is using the `RollingUpdate` deployment strategy, Strimzi will
+    1) Create an empty StrimziPodSet without any Pods
+    2) Add new Kafka Connect Pod to the StrimziPodSet
+    3) Wait for the new Pod to become ready
+    4) Scale-down the Deployment by one replica to remove one Kafka Connect Pod from the Deployment
+    5) ... repeat steps 2-4 until the StrimziPodSet has the desired number of Pods and the Deployment has 0 replicas
+    6) Delete the Deployment
+* If the user is using the `Recreate` deployment strategy, Strimzi will
+    1) Create an empty StrimziPodSet without any Pods
+    2) Scale-down the Deployment by one replica to remove one Kafka Connect Pod from the Deployment
+    3) Add new Kafka Connect Pod to the StrimziPodSet
+    4) Wait for the new Pod to become ready
+    5) ... repeat steps 2-4 until the StrimziPodSet has the desired number of Pods and the Deployment has 0 replicas
+    6) Delete the Deployment
+
+Doing the things in the right oder is important because otherwise it might happen that there will be no space in the cluster to start the new Pod and the migration might get stuck.
+
+### Downgrades
+
+The migration from StrimziPodSets to Kubernetes Deployments will be used in the following situations:
+* Downgrading from Strimzi versions with enabled `StableConnectIdentities` feature gate to Strimzi version with disabled `StableConnectIdentities` feature gate
+* Disabling of the `StableConnectIdentities` feature gate in Strimzi versions which supports it
+
+The migration will happen automatically and will use the following steps:
+* If the user is using the `RollingUpdate` deployment strategy, Strimzi will
+    1) Create a new Deployment with 0 replicas
+    2) Scale-up the Deployment by increasing the replica count by 1 to create a new Pod
+    3) Wait for the new Pod to become ready
+    4) Remove one Pod from the StrimziPodSet to get it removed
+    5) ... repeat steps 2-4 until the Deployment has the desired number of Pods and the StrimziPodSet has no Pods
+    6) Delete the StrimziPodSet
+* If the user is using the `Recreate` deployment strategy, Strimzi will
+    1) Create a new Deployment with 0 replicas
+    2) Remove one Pod from the StrimziPodSet to get it removed
+    3) Scale-up the Deployment by increasing the replica count by 1 to create a new Pod
+    4) Wait for the new Pod to become ready
+    5) ... repeat steps 2-4 until the Deployment has the desired number of Pods and the StrimziPodSet has no Pods
+    6) Delete the StrimziPodSet
+
+Doing the things in the right oder is important because otherwise it might happen that there will be no space in the cluster to start the new Pod and the migration might get stuck.
```

Comment:
```suggestion
Doing the things in the right order is important because otherwise it might happen that there will be no space in the cluster to start the new Pod and the migration might get stuck.
```